source/equipment/equipment_api.cpp

```diff
@@ -0,0 +1,140 @@
+/* Copyright © 2001-2019, Canal TP and/or its affiliates. All rights reserved.
+
+This file is part of Navitia,
+    the software to build cool stuff with public transport.
+
+Hope you'll enjoy and contribute to this project,
+    powered by Canal TP (www.canaltp.fr).
+Help us simplify mobility and open public transport:
+    a non ending quest to the responsive locomotion way of traveling!
+
+LICENCE: This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+Stay tuned using
+twitter @navitia
+IRC #navitia on freenode
+https://groups.google.com/d/forum/navitia
+www.navitia.io
+*/
+
+#include "equipment_api.h"
+#include "utils/paginate.h"
+
+#include "boost/range/algorithm.hpp"
+
+#include <tuple>
+#include <vector>
+#include <string>
+
+namespace navitia {
+namespace equipment {
+
+namespace {
+const std::string build_ptref_sa_filter(const std::string& line_uri, const std::string& filter) {
+    std::string sa_filter = "line.uri=" + line_uri;
+    if (filter.size()) {
+        sa_filter += " and (" + filter + ")";
+    }
+    return sa_filter;
+}
+
+const std::string build_ptref_sp_filter(const std::string& stop_area_uri, const std::string& filter) {
+    std::string sp_filter = "stop_area.uri=" + stop_area_uri;
+    if (filter.size()) {
+        sp_filter += " and (" + filter + ")";
+    }
+    return sp_filter;
+}
+
+void fill_equipment_to_pb(const equipment::EquipmentReportList& reports, PbCreator& pb_creator, int depth) {
```

Comment:
```suggestion
void fill_equipment_to_pb(const equipment::EquipmentReportList& reports, PbCreator& pb_creator, const int depth) {
```

Reply:
No need to be `const` as it's passed by copy ;)

Reply:
but your copy can be const in this part of code ?